src/include/btree.i

```diff
@@ -1543,6 +1545,48 @@ __wt_btree_lsm_over_size(WT_SESSION_IMPL *session, uint64_t maxsize)
 }
 
 /*
+ * __wt_btree_lsm_flip_primary --
+ *      Update the btree and cache dirty leave page values.
```

Comment:
Suggest: "Switch a btree handle to/from the current primary chunk of an LSM tree".